lib/routes/missav/new.ts

```diff
@@ -33,9 +34,9 @@ export const route: Route = {
 
 async function handler() {
     const baseUrl = 'https://missav.com';
-    const { data: response } = await got(`${baseUrl}/dm397/new`);
+    const browser = await puppeteer();
```

Comment:
Unnecessary use of puppeteer. The site can be accessed by passing [RSSHub's useragent](https://github.com/DIYgod/RSSHub/blob/6e5738e4dce8464e4b072ba59dc1d4a02f6fb036/lib/config.ts#L391).